types/polygon/polygon-tests.ts

```diff
@@ -0,0 +1,2 @@
+// no tests yet
+const ok = 1;
```

Comment:
Time to write some!

Reply:
I wrote some very simple tests to check the instance creation and the way the modules are imported. Is this ok?
I am not the author of the original library. Just wanted to provide typings for it.